usage.txt

```diff
@@ -11,38 +11,47 @@ Environment variables:
 
   AR:
       If no AR value is specified, the script will attempt to set AR to
-      llvm-ar-10 llvm-ar-9, llvm-ar-8, llvm-ar-7, llvm-ar, then
-      ${CROSS_COMPILE}ar if they are found in PATH.
+      llvm-ar, then ${CROSS_COMPILE}ar if they are found in PATH.
```

Comment:
I might word all of these as "the script will attempt to see <VAR> to <llvm_name> and fallback to ${CROSS_COMPILE}<gnu_name> if it is not found in PATH".

I know Nick was wanting to eliminate that behavior but still.

Otherwise, Harsh's changes also seem good to me.

Reply:
Actually this makes more sense than current loops in driver.sh, so maybe the logic there should be revisited.